INSTALL.md

```diff
@@ -101,13 +101,17 @@ Create config file, based on `dendrite-config.yaml`. Call it `dendrite.yaml`. Th
 ## Starting a monolith server
 
 It is possible to use 'naffka' as an in-process replacement to Kafka when using
-the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`.
+the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`. 
+
+In the `dendrite.yaml` file, one must also uncomment the necessary line (related to naffka) to use naffka.
```

Comment:
```suggestion
Additionally, uncomment the necessary line (related to naffka) under `database:`.
```

We can probably simplify this a bit to remove the double mention of `dendrite.yaml`. Also since this relates to the previous paragraph and it's not too long we can probably just merge the two into a single paragraph.

Reply:
Sure, ill combine them together